db_stress_tool/no_batched_ops_stress.cc

```diff
@@ -442,7 +442,7 @@ class NonBatchedOpsStressTest : public StressTest {
         if (!s.ok()) {
           fprintf(stderr, "dropping column family error: %s\n",
                   s.ToString().c_str());
-          std::terminate();
+          thread->shared->SafeTerminate();
```

Comment:
I still see a couple places in this file where `std::terminate()` is called. Is the reason they are not changed that `fault_fs_guard->PrintFaultBacktrace();` is called before terminate?

Reply:
Yes, I don't know if fault_fs_guard is thread safe